lib/util.ml

```diff
@@ -224,6 +224,36 @@ let search_in_path p s =
 
 let search_in_assets = search_in_path Secure.assets
 
+let hash_file file =
+  if Sys.file_exists file then
```

Comment:
You don't need to test the existence of the file. The function `open_in_bin` will fail with an exception.